### ISSUE TITLE
"h1 project token add" returns list

### ISSUE THREAD

**Issue by ad-m** [hyperonecom/h1-cli#67]
In the case of creating objects (resources), we usually return only this object. The presented method departs from this, becasue returns output as ```h1 project token list```.

**Comment by ad-m** [comment]
```project token access add ``` too.

**Comment by ad-m** [comment]
@fredericosilva , there is missing URL for token access & ingress still:

```
$ h1 firewall ingress show --firewall secure-zone-fw --ingress 5b3b52107109678548c342f9 --verbose
verbose: GET https://api.hyperone.com/v1//firewall/secure-zone-fw/ingress/5b3b52107109678548c342f9/5b3b52107109678548c342f9
error: 404 {"status":404}
$ h1 project token access show --token **CUT** --access 5b37d630ddd5b3c5e9fda91e --verbose
verbose: GET https://api.hyperone.com/v1/tenant/5af0bbbcb7802508ad844caa/credential/authtoken/**CUT**/access/5b37d630ddd5b3c5e9fda91e
error: 404 {"status":404}
```

**Comment by fredericosilva** [comment]
@ad-m vault credentials should have the same problem

**Comment by fredericosilva** [comment]
@ad-m fixed on the API side

**Comment by ad-m** [comment]
Fixed in tests in #111
